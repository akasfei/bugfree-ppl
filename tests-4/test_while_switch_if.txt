i, j;
{
  i = 3 ;
  j = 0;
  print j;
  print i;
  WHILE i > 0
  {
  SWITCH i
  {
    CASE 2: { j = 22; }
    CASE 3: { j = 33; }
    DEFAULT: {
      IF j > 10 { j = j - 10; }
    }
  }
  i = i - 1;
  print j;
  }
  print i;
}
