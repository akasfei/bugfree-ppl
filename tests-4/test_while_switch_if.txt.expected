0
3
33
22
12
0
